### PR TITLE
added entry for Sequel Ace

### PR DESCRIPTION
**Pull request by nogorilla**
0. [x] I have read the [Contribution Guidelines](https://github.com/iCHAIT/awesome-macOS/blob/master/.github/contributing.md)
1. [x] Project URL: https://github.com/Sequel-Ace/Sequel-Ace
2. [x] Why should this project be added: This project is a fork of the popular Sequel Pro app. However, Sequel Pro appears to be abandoned at this point with issues and errors prevent it from running. See this [issue](https://github.com/sequelpro/sequelpro/issues/3705)
3. [x] End all descriptions with a full stop/period
4. [x] Entry is ordered alphabetically
5. [x] Appropriate icon(s) added if applicable (OSS, freeware)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,8 +240,7 @@
 - [1Password](https://1password.com) - Password Manager and Secure Wallet.
 - [AnyBar](https://github.com/tonsky/AnyBar) - A menubar status indicator. [![Open-Source Software][OSS Icon]](https://github.com/tonsky/AnyBar) ![Freeware][Freeware Icon]
 - [APNGb](https://github.com/mancunianetz/APNGb) - .apng image assembler/disassembler app. [![Open-Source Software][OSS Icon]](https://github.com/mancunianetz/APNGb) ![Freeware][Freeware Icon]
-- [AppCleaner](http://freemacsoft.net/appcleaner/) - Uninstall your apps easily. ![Freeware][Freeware Icon]
-- [Arthur](http://arthur-osx.com/) - Safely optimize some of your Mac's hidden settings.
+- [AppCleaner](http://freemacsoft.net/appcleaner/) - Uninstall your
```
Comment (dkhamsing) [inline README.md:243]: Did you mean to change AppCleaner ?

Comment (nogorilla) [reply README.md]: I did not, thanks for the catch.

**Comment by nogorilla** [comment]
not including those additional changes caused failing tests. I can extract those changes into separate PRs but they will need to be approved first

**Comment by dkhamsing** [comment]
@nogorilla don't worry about the tests - but thanks if you'd like to fix them, please do not use the same pull request as your submission - thanks!

**Comment by dkhamsing** [comment]
Thanks @nogorilla @herrbischoff

**Pull request merged**